connectivity/check/metrics.go

```diff
@@ -53,7 +53,7 @@ func (a *Action) collectPrometheusMetrics(source MetricsSource) (promMetricsPerN
 }
 
 // collectMetricsForPod retrieves the metrics for one pod.
-func (a *Action) collectMetricsForPod(pod Pod, port string) (map[string]*dto.MetricFamily, error) {
```

Comment:
nit: This change here seems to be part of the wrong commit

Reply:
Oupsy! Fixed!